sdk/python/pvsite_datamodel/sqlmodels.py

```diff
@@ -106,7 +107,7 @@ class ForecastSQL(Base, CreatedMixin):
     forecast_version = sa.Column(sa.String(32), nullable=False)
 
     # one (forecasts) to many (forecast_values)
-    forecast_values = relationship("ForecastValueSQL")
+    forecast_values: List["ForecastValueSQL"] = relationship("ForecastValueSQL")
```

Comment:
nice

Reply:
Fiddled with mypy for a bit and then got too confused!